lasagne/layers/special.py

```diff
@@ -179,7 +181,7 @@ def get_output_for(self, input, **kwargs):
         return input * self.scales.dimshuffle(*pattern)
 
 
-def standardize(layer, offset, scale, shared_axes):
+def standardize(layer, offset, scale, shared_axes='auto'):
```

Comment:
This was actually part of another PR that you may have accidentally copied in... seems it doesn't impede mergeability, but maybe it's better to remove. Don't know exactly if it will change the history in any way. Please wait until you read my upcoming comment on squashing commits, though :)